spec/requests/stripe_model_callbacks/error_handling_spec.rb

```diff
@@ -9,8 +9,8 @@
     configuration.instance_variable_set(:@on_error_callbacks, [])
   end
 
-  it "creates an invoice" do
-    expect(StripeModelCallbacks::Invoice::UpdatedService).to receive(:reported_execute!).and_raise("BOOM!")
+  it "creates an invoice", :aggregate_failure do
```

Comment:
What is "aggregate_failure" used for?

Reply:
To group `expect` failures and allow to run second expect when first is failing.

Reply:
I forgot to remove it, need that to debug.

Reply:
```suggestion
  it "creates an invoice" do
```